src/QsCompiler/CommandLineTool/Commands/Diagnose.cs

```diff
@@ -56,6 +56,10 @@ public static IEnumerable<Example> UsageExamples
             [Option("code", Required = false, Default = false,
             HelpText = "Specifies whether to print the Q# code generated based on the built syntax tree.")]
             public bool PrintCompiledCode { get; set; }
+
+            [Option("trim", Required = false, Default = false,
+            HelpText = "Specifies whether to trim the syntax tree to eliminate selective abstractions.")]
```

Comment:
I'm not sure I understand what this option does from the `HelpText`, sorry. As a side question, is it possible to expose this through an MSBuild property?

Reply:
The trimming currently inlines all conjugations. This will need to be revised and properly integrated with the whole setup for targeting later, so this is mostly temporary and the handle only exists for the "diagnose" verb (that is specifically for debugging the compiler really). For building, then trimming is currently always done.